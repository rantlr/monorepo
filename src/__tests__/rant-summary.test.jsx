import * as React from 'react';
import { render, cleanup } from 'react-testing-library';
import RantSummary from '../rant-summary';

const createdAt = '2018-11-17T15:19:52.675Z';
const updatedAt = '2018-12-23T10:11:12.131Z';

afterEach(cleanup);

test('snapshot', () => {
  render(
    <RantSummary
      id="1234"
      title="Some title"
      background="Some background"
      created_at={createdAt}
      updated_at={updatedAt}
    />,
  );

  expect(document.documentElement).toMatchSnapshot();
});
