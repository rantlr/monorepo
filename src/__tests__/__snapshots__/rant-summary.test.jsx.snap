// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<html>
  <head />
  <body>
    <div>
      <article
        id="Some%20title"
      >
        <a
          href="/rant/1234"
        >
          <h1>
            Some title
          </h1>
        </a>
        <p>
          Some background
        </p>
        <p>
          Created at 
          16:19:52 on Saturday November 17th 2018
        </p>
        <p>
          Updated at 
          11:11:12 on Sunday December 23th 2018
        </p>
      </article>
    </div>
  </body>
</html>
`;
